allensdk/brain_observatory/behavior/swdb/behavior_project_cache.py

```diff
@@ -138,21 +130,21 @@ def get_session(self, experiment_id):
 
     def get_container_sessions(self, container_id):
         container_stages = {}
-        container_manifest = self.manifest.groupby('container_id').get_group(container_id)
-        for ind_row, row in container_manifest.iterrows():
+        container_experiments = self.experiment_table.groupby('container_id').get_group(container_id)
+        for ind_row, row in container_experiments.iterrows():
             container_stages.update(
                 {row['stage_name']: self.get_session(row['ophys_experiment_id'])}
             )
         return container_stages
 
```

Comment:
It looks like this method has been commented out entirely. What's the story here?

Reply:
i removed this in recent push because the new version works